src/style.css

```diff
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.moyo-card {
```

Comment:
use only `card `and `stars `blocks in this task. And place them in separate files